live/app/components/follower-form.js

```diff
@@ -71,7 +71,7 @@ export default Ember.Component.extend({
 
       const store = this.get('store');
       const follower = store.createRecord('follower', {email: email});
-      follower.save()
+      this.attrs.save(follower)
```

Comment:
Ce n'est pas mieux de faire `this.get('save')` plutôt que `this.attrs.save` ? accéder directement aux attributs n'est-il pas trop bas niveau ? ou alors on s'en fiche car on n'a pas d'écouteur sur la propriété `save` ?